src/js/tech/tech.js

```diff
@@ -33,10 +34,20 @@ import document from 'global/document';
 class Tech extends Component {
 
   constructor(options={}, ready=function(){}){
+    let defaults = {
```

Comment:
I don't quite understand what this object is used for. How is this different from the `features` thing?